examples/proc_raw.py

```diff
@@ -18,6 +18,12 @@
 ``py proc_raw.py CPMG_9 240620_200uM_TEMPOL_pm_generic_CPMG.h5
 ODNP_NMR_comp/Echoes``
 
+# PR -- this field sweep is not good field sweep
+# data.  (1) The acquisition length is small.  (2)
+# Phase cycling noise and difference in signal
+# amplitude across nScans indicates bad repetition
+# delay.  (3) You can't see the rise and fall of
+# the ESR resonance.
```

Comment:
please see and deal with comment

Reply:
replaced with wider and better resolved with newer data